backend/src/models/preferencesModel.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+
+const PreferencesSchema = new mongoose.Schema({
+    userID: { type: mongoose.Schema.Types.ObjectId, required: true },
+    minPrice: {type: Number, default: 0, required: true},
+    maxPrice: {type: Number, required: true},
+    housingType: {type: String, enum: ['studio', '1-bedroom', '2-bedroom', 'other'], required: true},
+    roommateCount: {type: Number, required: true},
+    petFriendly: { type: Boolean, required: true },
+    smoker: {type: String, enum: ['non-smoker', 'socially', 'regular'], required: true},
```

Comment:
Should smoker have `neutral` instead of `socially`? And `regular` match up with the other ones just for consistent API?

Reply:
Sure! Can change this accordingly! Was not sure what the best wording was for smoker actually